com.makemkv.MakeMKV.appdata.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.makemkv.MakeMKV</id>
+  <launchable type="desktop-id">com.makemkv.MakeMKV.desktop</launchable>
+  <name>MakeMKV</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>DVD and Blu-ray to MKV converter and network streamer</summary>
+  <description>
+    <p>
+      MakeMKV is your one-click solution to convert video that you own
+      into free and patents-unencumbered format that can be played
+      everywhere. MakeMKV is a format converter, otherwise called
+      "transcoder". It converts the video clips from proprietary (and
+      usually encrypted) disc into a set of MKV files, preserving most
+      information but not changing it in any way. The MKV format can
+      store multiple video/audio tracks with all meta-information and
+      preserve chapters.
+    </p>
+    <p>
+      Additionally MakeMKV can instantly stream decrypted video
+      without intermediate conversion to wide range of players, so you
+      may watch Blu-ray and DVD discs with your favorite player on
+      your favorite OS or on your favorite device.
+    </p>
+    <p>NOTE: users of this software must accept the terms of the "End
```

Comment:
If user is supposed to accept anything prior starting the app, either add some simple wrapper or ask if application itself could do so.